packaging/windows/build_pypi_pkg/generate_win_pypi_pkg.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+TORCH_VER=$1
+SETUP_FILE="setup.py"
+REPLACE_KEYWORD="{{GENERATE_TORCH_PKG_VER}}"
+
+mkdir torch
+cp $SETUP_FILE torch/
+cd torch
+sed -i "s/$REPLACE_KEYWORD/$TORCH_VER/g" $SETUP_FILE
+python $SETUP_FILE sdist
```

Comment:
If this is a Windows only package, maybe you should provide more args otherwise it will generate a source package for all platforms.

Reply:
More details about this is written in [PEP-425](https://www.python.org/dev/peps/pep-0425/#id14).

Reply:
I did a google search and was amazed to find out that it is currently impossible to generate a platform-specific sdist package. See https://www.python.org/dev/peps/pep-0625/ and https://docs.python.org/3/distutils/sourcedist.html .

Reply:
Yep, I just want to find sdist options, does not find one.